src/Error/Middleware/ErrorHandlerMiddleware.php

```diff
@@ -216,6 +216,9 @@ protected function handleInternalError(): ResponseInterface
     protected function getErrorHandler(): ErrorHandler
     {
         if ($this->errorHandler === null) {
+            deprecationWarning(
+                'Using an `errorHandler` is deprecated. You should migate to the `ExceptionTrap` sub-system instead.'
```

Comment:
```suggestion
                'Using an `ErrorHandler` is deprecated. You should migate to the `ExceptionTrap` sub-system instead.'
```
We want this to be the class name, right?